compendium/DeclarativeServices/src/manager/ComponentConfigurationImpl.hpp

```diff
@@ -56,7 +56,7 @@ namespace cppmicroservices
         {
             ListenerToken(std::string pid, const ListenerTokenId tokenId)
                 : pid(std::move(pid))
-                , tokenId(std::move(tokenId))
```

Comment:
should the parameter be non-const? is it better to move ` tokenId`?

Reply:
Since ListenerTokenId is only a uint64, it's trivial to copy, so no move is necessary.
For more complex types, however, it would be better to remove the const and keep the move.